### PR TITLE
Added comment 2

### PR DESCRIPTION
**Pull request by kittdotech**
**Changes:**

- Added comment
- Updated test cases

### CODE REVIEW

File: add.php [hunk 1]
```diff
@@ -4,6 +4,7 @@
    include "cn.php";
 
     // Check connection 1
+   // Check connection 2
```
Comment (Deiveegan) [inline add.php]: modify comment 2,3

File: add.php [hunk 2]
```diff
@@ -4,6 +4,7 @@
    include "cn.php";
```
Comment (Deiveegan) [inline add.php:4]: add headder ty

**Review by Deiveegan — CHANGES_REQUESTED** [review]

File: add.php (hunk 2 above)
Comment (kittdotech) [reply add.php:4]: Already its included

File: add.php (hunk 1 above)
Comment (kittdotech) [reply add.php]: I will change

**Review by Deiveegan — APPROVED** [review]

**Pull request merged**